addons/mail/static/src/js/composer.js

```diff
@@ -557,8 +557,12 @@ var BasicComposer = Widget.extend(chat_mixin, {
                 break;
             // ESCAPE: close mention propositions
             case $.ui.keyCode.ESCAPE:
-                event.stopPropagation();
-                this.mention_manager.reset_suggestions();
+                if (this.mention_manager.is_open()) {
+                    event.stopPropagation();
+                    this.mention_manager.reset_suggestions();
+                } else {
+                    this.trigger("keyup_escape");
```

Comment:
also, the name of the event is not so good.  rename keyup_escape into 'escape_pressed'